### ISSUE TITLE
Can you make this plug-in gradle supported

### ISSUE THREAD

**Issue by xjl456852** [krasa/MavenHelper#64]
Can you make this plug-in gradle supported
thank you

**Comment by krasa** [comment]
you mean this https://github.com/JetBrains/gradle-intellij-plugin ?

**Comment by xjl456852** [comment]
> you mean this https://github.com/JetBrains/gradle-intellij-plugin ?

no, I want use MavenHelper in gradle project, not only use it in maven project

**Comment by krasa** [comment]
It would be nice, but I do not plan to do it. 
Pull request welcomed.
